src/lib/components/Tree.vue

```diff
@@ -20,15 +20,15 @@
         @delete-row="onDeleteRow"
         @node-click="onNodeClick"
         @node-expanded="onNodeExpanded"
-        @toggle-checkbox="toggleCheckbox"
+        @toggle-checkbox="onToggleCheckbox"
       >
         <template #checkbox="{ node: slotNode, checked, indeterminate }">
           <slot
             name="checkbox"
             :node="slotNode"
             :checked="checked"
             :indeterminate="indeterminate"
-            :toggleCheckbox="() => toggleCheckbox(slotNode)"
+            :toggleCheckbox="() => onToggleCheckbox(slotNode)"
```

Comment:
We can also correct attr name as `toggle-checkbox`

Reply:
This is the API offered by the package. Changing it may cause incompatibilities.
https://teamseodo.github.io/vue3-tree/guide/slots.html#checkbox

Reply:
Now I understand the part you highlighted. 👍🏻